app/hooks/use-stand-polling.ts

```
@@ -1,55 +1,140 @@
-      const res = await fetch(`/api/stands/status?sectorId=${sectorId}`);
-      if (res.ok) {
-        const data = await res.json();
-        onUpdateRef.current(data.stands);
-      }
-    } catch (error) {
-      console.error("Stand polling error", error);
+  const [stale, setStale] = useState(false);
+  const inFlightRef = useRef(false);
+  const appliedVersionRef = useRef(0);
+  const lastSuccessAtRef = useRef<number | null>(null);
+  const startedAtRef = useRef<number>(Date.now());
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/app/hooks/use-stand-polling.ts:30:39
  28 |   const appliedVersionRef = useRef(0);
  29 |   const lastSuccessAtRef = useRef<number | null>(null);
> 30 |   const startedAtRef = useRef<number>(Date.now());
     |                                       ^^^^^^^^^^ Cannot call impure function
  31 |   const failureCountRef = useRef(0);
  32 |   const trackerRef = useRef(new LatestRequest());
  33 |

(react-hooks/purity)